src/components/Totals.js

```diff
@@ -31,10 +37,11 @@ export class Totals extends Component {
   }
 
   render() {
+    const items = this.state.desktop ? this.state.items : this.state.items.slice(0, 2);
```

Comment:
it may be worth to destructure in the render function `render(_, {desktop, items})`,  we can cut down on the `this.` quite a bit on this line

Reply:
I actually prefer using this over all the implicit properties, but destructuring is the common practice so we can do that

Reply:
fair enough, either is fine, whicever you prefer 👍